sql/core/src/test/scala/org/apache/spark/sql/connector/SimpleWritableDataSource.scala

```diff
@@ -41,11 +41,9 @@ import org.apache.spark.util.SerializableConfiguration
  * Each task writes data to `target/_temporary/uniqueId/$jobId-$partitionId-$attemptNumber`.
  * Each job moves files from `target/_temporary/uniqueId/` to `target`.
  */
-class SimpleWritableDataSource extends SimpleTableProvider with SessionConfigSupport {
+class SimpleWritableDataSource extends SimpleTableProvider {
```

Comment:
can we extend `TestingV2Source` instead?

Reply:
yes